drivers/can/can_mcan.h

```diff
@@ -178,8 +178,10 @@ struct can_mcan_data {
 	void *cb_arg_ext[NUM_EXT_FILTER_DATA];
 	can_state_change_callback_t state_change_cb;
 	void *state_change_cb_data;
+	uint32_t std_filt_fd_frame;
 	uint32_t std_filt_rtr;
 	uint32_t std_filt_rtr_mask;
+	uint8_t ext_filt_fd_frame;
```

Comment:
I stumbled upon the difference in size for `std` and `ext` filters and found there is a potential variable overflow bug, see #53417. This should be `uint16_t`.

I have created a separate PR to fix the other two flags: #53418

Reply:
> I stumbled upon the difference in size for `std` and `ext` filters and found there is a potential variable overflow bug, see #53417. This should be `uint16_t`.
> 
> I have created a separate PR to fix the other two flags: #53418

@henrikbrixandersen @martinjaeger  I updated size for `ext_filt_fd_frame`

Reply:
Could you pls help me review this PR https://github.com/zephyrproject-rtos/zephyr/pull/52450.